### PR TITLE
PIM-5694: Add support for real numbers (not as strings) when importing XLSX files

### PR DESCRIPTION
**Pull request by phaseinducer**
Same as https://github.com/akeneo/pim-community-dev/pull/4284 but for numbers cast to integers/floats.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n/a
| Added Behats                      | yes
| Changelog updated                 | n/a
| Review and 2 GTM                  | yes
| Micro Demo to the PO (Story only) | 
| Migration script                  | n/a
| Tech Doc                          |

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

**Comment by jjanvier** [comment]
gtm

File: features/import/product/xlsx/import_products_with_numbers.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@javascript
+Feature: Import XLSX products with dates
```
Comment (solivier) [inline features/import/product/xlsx/import_products_with_numbers.feature]: with numbers ?

Comment (phaseinducer) [inline features/import/product/xlsx/import_products_with_numbers.feature]: yep, good catch

**Pull request merged**